services/metadata/src/metadata/routes/templates.clj

```diff
@@ -57,7 +61,14 @@
         :description "This endpoint returns a CSV file guide for a specific template.
                      It's intended to be downloaded and used as a reference while
                      filling out a file from the blank-csv endpoint for the same template."
-        (csv-download-resp attachment "guide.csv" (templates/view-template-guide template-id))))))
+        (csv-download-resp attachment "guide.csv" (templates/view-template-guide template-id)))
+
+      (GET* "/zip-csv" []
+        :query [{:keys [attachment]} CSVDownloadQueryParams]
+        :summary "Get a ZIP file containing both the guide and blank CSV for a template."
+        :description "This endpoint returns a zip file containing both the
+                     blank CSV and the guide CSV for the same template."
+        (filetype-download-resp "application/zip" attachment "template.zip" (templates/view-template-zip template-id))))))
```

Comment:
The changes in this file, aside from the obvious adding of a new endpoint, are just to make it so we can return the proper `application/zip` MIME type.